modules/navier_stokes/src/fvbcs/INSFVTKEDWallFunctionBC.C

```diff
@@ -26,7 +26,8 @@ INSFVTKEDWallFunctionBC::validParams()
   params.addRequiredParam<MooseFunctorName>(NS::density, "Density");
   params.addRequiredParam<MooseFunctorName>(NS::mu, "Dynamic viscosity.");
   params.addRequiredParam<MooseFunctorName>(NS::mu_t, "The turbulent viscosity.");
-  params.addRequiredParam<MooseFunctorName>(NS::TKE, "The turbulent kinetic energy.");
```

Comment:
Please try to only use variable names in the NS namespace

Reply:
see first comment, this is using NS but NS got changed so we have to code "k" as the deprecated parameter